app/src/main/java/com/gruppo4/SMSApp/MainActivity.java

```diff
@@ -0,0 +1,140 @@
+package com.gruppo4.SMSApp;
+
+/**
+ * @author Francesco Bau'
+ */
+
+import com.gruppo4.audioUtility.AudioUtilityManager;
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.os.Bundle;
+import android.os.Vibrator;
+import android.view.View;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+
+public class MainActivity extends AppCompatActivity {
+
+
+    // Default app volume, expressed in percentage
+    private static final int DEFAULT_APP_VOLUME = 50;
+    // The selected Stream.
+    private static AudioUtilityManager.AUMStream stream;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        //Default choice: Stream ALARM
+        stream = AudioUtilityManager.AUMStream.ALARM;
+
+        // Action performed by clicking button0
+        // -> TEST the method getVolume(Context,String).
+        // -> TEST the method setVolume(Context,String,int).
+        findViewById(R.id.button0).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                // Sets up the volume at default percentage.
+                AudioUtilityManager.setVolume(MainActivity.this, stream, DEFAULT_APP_VOLUME);
+                int defaultVolume = AudioUtilityManager.getVolume(MainActivity.this, stream);
+                Toast.makeText(MainActivity.this, "DEFAULT " + stream + " VOLUME: " + defaultVolume + " %.", Toast.LENGTH_SHORT).show();
+            }
+        });
+
+        // Action performed by clicking button1
```

Comment:
In general, when more than a couple buttons are present in an UI it would be advisable to add a brief comment explaining what the action of the button is. Such actions are clear when you read the code, but not having a comment makes situations like "find the button that does x" harder, especially with no name.

Reply:
The description of the action is right below it, in a new line.

Reply:
Oh I see what do you mean.. I had to be more specific, like saying "Action performed by clicking button0 -> set Volume to Default", and same for the other buttons?

Reply:
Exactly, it would be clearer to have the whole explanation all together

Reply:
Ok, thank you!